org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug348199TestLanguageGrammarAccess.java

```diff
@@ -259,8 +259,10 @@
 		//Timeperiod:
 		//	{Timeperiod}
 		//	'define' 'timeperiod' ('1'
-		//	'{' (NL+ time+=TimeDef1)* (NL+ 'timeperiod_name' name+=ID (NL+ time+=TimeDef2)* & (NL+ 'otherelement' alias+=ID (NL+
-		//	time+=TimeDef3)*)?) NL+
+		//	'{' (NL+ time+=TimeDef1)* ( NL+ 'timeperiod_name' name+=ID (NL+ time+=TimeDef2)* & (NL+ 'otherelement' alias+=ID (NL+
+		//	time+=TimeDef3)*)?
+		//		
```

Comment:
Blank line?

Reply:
We preserve most newlines because they might be intentionally there. Might be annoying for the user if we remove the newlines inside his rules.